### PR TITLE
Prevent 500 error when staff tries to see XBlock unit content as specific student

### PR DESCRIPTION
**Pull request by amirtds**

### CODE REVIEW

**Comment by amirtds** [comment]
[(4) NYIF: server error message caused by XBlocks when viewing as a specific student](https://trello.com/c/ZTUHFEid/3994-4-nyif-server-error-message-caused-by-xblocks-when-viewing-as-a-specific-student)

**Review by bryanlandia — COMMENTED** [review]
looks good!  I think you said this one didn't look like it was being maintained upstream, but still would be worth opening a PR

**Comment by amirtds** [comment]
@bryanlandia Yes I tried to open PR but I got permission denied error seems opening PR is closed.

File: pdf/pdf.py [hunk 1]
```diff
@@ -38,17 +41,17 @@ class pdfXBlock(XBlock):
         default=_("http://tutorial.math.lamar.edu/pdf/Trig_Cheat_Sheet.pdf"),
         scope=Scope.content,
         help=_("The URL for your PDF."))
-    
+
```
Comment (OmarIthawi) [inline pdf/pdf.py]: Please put those space changes in a separate commit, or just revert them.

**Review by OmarIthawi — CHANGES_REQUESTED** [review]
Thanks @amirtds, please let me know if you've tested the changes on an instance that I can verify.

Also, @bryanlandia for the XBlock, this _is_ the upstream pull request. The other repo is no longer maintained as of about a year ago (see https://github.com/MarCnu/pdfXBlock/pull/6).

**Comment by amirtds** [comment]
Thanks, @OmarIthawi Yes this is pushed to NYIF staging server

File: pdf/pdf.py (hunk 1 above)
Comment (amirtds) [reply pdf/pdf.py]: @OmarIthawi I removed the space changes for now.

**Review by OmarIthawi — APPROVED** [review]
LGTM, thanks @amirtds!
